testplan/report/testing/base.py

```diff
@@ -66,13 +68,17 @@ class Status(object):
     INCOMPLETE = 'incomplete'
     PASSED = 'passed'
     SKIPPED = 'skipped'
+    READY = 'ready'
+    RUNNING = 'running'
 
     STATUS_PRECEDENCE = (
+        RUNNING,
         ERROR,
         FAILED,
         INCOMPLETE,
         PASSED,
```

Comment:
Not sure about READY having the lowest precedence, so if one testcase PASSES and other entries are not run yet, the entire report will show as PASSED?

Reply:
READY has the highest precedence.

Reply:
Now I think about it, maybe READY should be a higher priority than PASSED/FAILED. That way, interactive testplan will only be green when all testcases have passed at least once.